Lib/test/test_call.py

```diff
@@ -492,6 +494,8 @@ def testfunction_kw(self, *, kw):
 
 class TestPEP590(unittest.TestCase):
```

Comment:
I think you could probably skip this whole class, since we don't have support for anything like vectorcalls yet.